### DISCUSSION TITLE
Améliorer l'expérience utilisateur avec les fichiers Lottie dans vos projets Flutter mobile et web

### DISCUSSION THREAD

**Discussion by giscus** [zero-filtre/zerofiltre-blog#151] [Announcements]
# Améliorer l'expérience utilisateur avec les fichiers Lottie dans vos projets Flutter mobile et web

Guide pratique qui explique comment ajouter des animations à des projets Flutter à l'aide de fichiers Lottie.

https://zerofiltre.tech/articles/232

**Comment by imphilippesimo** [comment]
Très bel Article, Bravo l'artiste 💪
